src/l500/l500-options.cpp

```diff
@@ -36,23 +42,39 @@ namespace librealsense
         return _range;
     }
 
+    float l500_hw_options::query_default( bool & success ) const
+    {
+        if(_fw_version >= firmware_version( MIN_GET_DEFAULT_FW_VERSION ) )
+        {
+            return query_new_fw_default( success );
+        }
+        else
+        {
+            return query_old_fw_default();
```

Comment:
Not good... `success` remains as it was...

Reply:
fixed